### DISCUSSION TITLE
运行一段时间就会出现panic：runtime error：integer divide by zero报错然后崩掉

### DISCUSSION THREAD

**Discussion by Fskyp9** [FloatTech/ZeroBot-Plugin#581] [Q&A] [answered]
![Z2U104PP XVD`N0GFXGKUQ6](https://user-images.githubusercontent.com/111845977/216989329-4e398c14-9bc9-4699-ba7d-680336587f67.jpg)
如图所示，运行一段时间就会出现类似错误，即使未下达任何指令与接受任何消息，此报错出现于更新1.6.2-beta1之后，之后版本都会出现此错误，此外，我在之前版本有按照readme中的示例设置过job：“记录在"@every 10s"触发的指令             拉取b站推送”，在之前的版本中会出现该job的信息
![image](https://user-images.githubusercontent.com/111845977/216996600-d06910cc-3eed-44c6-b21d-e84984a9cebd.png)
而在后面版本从开始运行到出现上述报错崩掉之前，从未出现过该job的信息，怀疑与job有关.
下面是部署用的设备信息
![image](https://user-images.githubusercontent.com/111845977/216994392-df58d1f1-1936-4ed8-bb31-0a72da681460.png)

**Comment by fumiama** [comment]
截图全一点，从运行开始到报错。

**Reply by Fskyp9** [reply]
![image](https://user-images.githubusercontent.com/111845977/217150680-3caebdb9-42eb-4f19-83cc-40ee0ed47aa6.png)
![image](https://user-images.githubusercontent.com/111845977/217150694-6f2af22d-8630-4246-b569-66825dd9d0a1.png)

**Comment by Fskyp9** [comment]
何でもない
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "FloatTech/ZeroBot-Plugin"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年2月7日(星期二) 中午11:59
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [FloatTech/ZeroBot-Plugin] 运行一段时间就会出现panic：runtime error：integer divide by zero报错然后崩掉 (Discussion #581)





 
截图全一点，从运行开始到报错。

**Comment by fumiama** [comment, marked as answer]
我似乎知道原因了，试试最新action版。

**Reply by Fskyp9** [reply]
问题解决了，没有再出现过这个报错了，赞
